libtiledbsoma/src/utils/arrow_adapter.cc

```diff
@@ -350,7 +350,13 @@ ArraySchema ArrowAdapter::tiledb_schema_from_arrow_schema(
                     dim_zstd_filter.set_option(
                         TILEDB_COMPRESSION_LEVEL,
                         platform_config.dataframe_dim_zstd_level);
-                } else if (soma_type == "SOMASparseNDArray") {
+                } else if (
+                    soma_type == "SOMASparseNDArray" ||
+                    soma_type == "SOMADenseNDArray") {
+                    // sparse_nd_array_dim_zstd_level being used for
```

Comment:
Sorry @nguyenv , no -- this is not at all what I meant earlier.

* What I thought you meant was that we could/should have a single `platform_config.dim_zstd_level` which would be used for dataframe, sparse nd array, and dense nd array
  * And my answer was: no, we need separate configs per datatype
* What I was advocating for (however inarticulate I may have succeeded in being, despite my best efforts) was:
  * that we _do_ need separate:
    * `platform_config.dataframe_dim_zstd_level`
    * `platform_config.sparse_nd_array_dim_zstd_level`
    * `platform_config.dense_nd_array_dim_zstd_level`
  * that our not having had a `platform_config.dense_nd_array_dim_zstd_level` was an oversight that needs to be remedied

Reply:
OK understood - I can change this.